srf_generation/input_file_generation/realisation_to_srf.py

```diff
@@ -433,6 +436,12 @@ def create_multi_plane_srf(
     clon = [parameter_dictionary.pop(f"clon_subfault_{i}") for i in range(plane_count)]
     clat = [parameter_dictionary.pop(f"clat_subfault_{i}") for i in range(plane_count)]
 
+    for key in list(parameter_dictionary.keys()):
+        # Remove all the subfault keys so they don't get passed to the sim_params.yaml file
+        # Must make a copy of the keys to be able to remove them.
+        if "_subfault_" in key:
+            parameter_dictionary.pop(key)
+
     # gets
     vel_mod_1d = parameter_dictionary.get(
```

Comment:
This part should also be changed too #scopecreep. That's the vel_mod_1d used for source generation getting propogated into the sim_params.yaml I was mentioning the other day too.